1-WebApp-OIDC/1-1-MyOrg/README.md

```diff
@@ -164,7 +164,8 @@ cd "1-WebApp-OIDC\1-1-MyOrg"
      by this line:
 
      ```CSharp
-      services.AddSignIn(Configuration);
+      services.AddAuthentication(OpenIdConnectDefaults.AuthenticationScheme)
+              .AddSignIn("AzureAd", Configuration, options => Configuration.Bind("AzureAd", options));
```

Comment:
@TiagoBrenck:
Why do we need both the section and the delegate.
This should be one or the other?

Reply:
cc: @jennyf19 @pmaytak

Reply:
The section is to configure `MicrosoftIdentityOptions.cs`. 

The delegate is the `Action<OpenIdConnectOptions>` where developers would be working with. With this, we are fully leveraging OpenIdConnect.